src/functionalTest/resources/features/F-703 - getCasePaymentOrder/F-703.feature

```diff
@@ -0,0 +1,127 @@
+@F-703
+Feature: GET Case Payment Order Endpoint
+
+  Background:
+    Given an appropriate test context as detailed in the test data source,
+    And a user with [an active profile in CCD],
+
+  @S-703.1
+  Scenario: AC1: Successfully allow the collection of existing payment order(s) from the Case Payment Order database (Happy Path 1 - Ids supplied in parameters)
+    And a successful call [to create a case payment order CP1] as in [Prerequisite_Create_CPO],
+    And a successful call [to create a case payment order CP2] as in [Prerequisite_Create_CPO_2],
+    When a request is prepared with appropriate values
+    And the request [contains a set of ids from the case payment orders previously created]
+    And the request [contains all the mandatory parameters]
+    And the request [does not contain defined optional parameters for page_size And page_number]
+    And it is submitted to call the [getCasePaymentOrder] operation of [Case Payment Orders Microservice]
+    Then a positive response is received
+    And the response [contains a 200 success OK code]
+    And the response [case payment order CP1, CP2]
+    And the response [has page_size default 20 and page_number default 1]
+    And the response has all other details as expected
+
+  @S-703.2
+  Scenario: AC2: Successfully allow the collection of existing payment order(s) from the Case Payment Order database (Happy Path 2 - Case Ids supplied in parameters)
+    And a successful call [to create a case payment order CP1] as in [Prerequisite_Create_CPO],
+    And a successful call [to create a case payment order CP2] as in [Prerequisite_Create_CPO_2],
+    When a request is prepared with appropriate values
+    And the request [contains a set of case ids from the case payment orders previously created]
+    And the request [contains all the mandatory parameters]
+    And the request [does not contain defined optional parameters for page_size And page_number]
+    And it is submitted to call the [getCasePaymentOrder] operation of [Case Payment Orders Microservice]
+    Then a positive response is received
+    And the response [contains a 200 success OK code]
+    And the response [case payment order CP1, CP2]
+    And the response [has page_size default 20 and page_number default 1]
+    And the response has all other details as expected
+
+  @S-703.3
+  Scenario: AC3: Mandatory parameters missing from the request (Search Criteria Missing)
+    And a successful call [to create a case payment order CP1] as in [Prerequisite_Create_CPO],
+    And a successful call [to create a case payment order CP2] as in [Prerequisite_Create_CPO_2],
```

Comment:
Do we need to create these case payment orders ?

Reply:
cpo's removed